lib/caps/MTL/d3d11

```diff
@@ -13,39 +13,42 @@ caps = dict(
   decode  = dict(
     avc     = dict(maxres = res4k , fmts = ["NV12"]),
     mpeg2   = dict(maxres = res2k , fmts = ["NV12"]),
-    jpeg    = dict(maxres = res16k, fmts = ["NV12", "411P", "422H", "422V", "444P", "Y800"]),
+    jpeg    = dict(maxres = res16k, fmts = ["NV12", "411P", "422H", "422V", "444P", "Y800", "IMC3", "BGRP", "400P", "YUY2", "UYVY", "ARGB"]),
     hevc_8  = dict(
       maxres    = res16k,
       fmts      = ["NV12", "YUY2", "AYUV"],
       features  = dict(scc = True, msp = True),
     ),
     hevc_10 = dict(maxres = res16k , fmts = ["P010", "Y210", "Y410"]),
     hevc_12 = dict(maxres = res16k , fmts = ["P012", "Y212", "Y412"]),
+    vp8     = dict(maxres = res4k , fmts = ["NV12"]),
```

Comment:
Vp8 decode support on MTL windows ? passed on linux but all failed Win test ffmpeg-libvpl on d3d11.

Reply:
Sync with Sherry lin, Vp8 decode not support on windows MTL